crm_department/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
That's a template, and it's a recommendation at best, not a requirement on itself.
The files are not even new, they are ported code.
So I don't see the problem with keeping the original headers.